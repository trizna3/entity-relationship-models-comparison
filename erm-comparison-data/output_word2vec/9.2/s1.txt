tile-positioning
weather-weather
fuel type-fuel type
units-units
boards-boards
games-games
tile type-tile types
CHANGE_CARDINALITY:S-boards-units
MOVE_ATTR_TO_INCIDENT_ENTITY_SET:S-health-boards
CHANGE_CARDINALITY:S-boards-units
MOVE_ATTR_TO_INCIDENT_ENTITY_SET:S-points-boards
REBIND_MN_TO_1NN1:S-boards-weather types
REBIND_MN_TO_1NN1:S-boards-tile types
