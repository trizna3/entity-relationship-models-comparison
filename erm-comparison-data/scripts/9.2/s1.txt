entity set games {
};

entity set "boards" {
	"rows"
	"columns"
};

entity set "tile types" {
};

entity set "weather types" {
};

entity set units {
	"name"
	"attack strength"
	"defense strength"
};

entity set "fuel type" {
};

association "actual state" between 
{
	games MANY
	boards ONE
}{
	"turn"
};

association "weather" between 
{
	boards MANY
	"weather types" MANY
}{
	row
	column
};

association "state" between 
{
	boards MANY
	units MANY
}{
	row
	column
	health
	points
};

association "positioning" between 
{
	boards MANY
	"tile types" MANY
}{
	row
	column
};


association between 
{
	units MANY
	"fuel type" ONE
}{};

association between 
{
	"weather types" MANY
	"tile types" MANY
	"fuel type" MANY
}{};