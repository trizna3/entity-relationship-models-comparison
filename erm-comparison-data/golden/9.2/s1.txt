games-games
boards-boards
tile-positioning
tile type-tile types
weather-weather types
fuel type-fuel type
units-units
REBIND_MN_TO_1NN1:S-boards-tile types
MOVE_ATTR_TO_INCIDENT_ENTITY_SET:S-turn-games